keepassxc-browser/offscreen/offscreen.js

```diff
@@ -0,0 +1,9 @@
+chrome.runtime.onMessage.addListener(({ target, action }, sender, sendResponse) => {
```

Comment:
I wasn't able to get this working with the browser-polyfill. Maybe I did it wrong or it didn't load in time, or something. I didn't spend a lot of time on it.

Reply:
https://github.com/mozilla/webextension-polyfill/issues/329 This issue thread didn't help either. I'll take a look when I the time.

Reply:
I don't know if the polyfill is that useful anymore since Chrome supports promises for most (if not all) APIs, and Firefox has always supported `chrome` instead of `browser`. 🤷 

Unless there's another reason for the polyfill that I have missed?

Reply:
The polyfill was used mostly because of the differences between promises and callbacks. I could investigate if we can get rid of the whole polyfill library for good :)

Reply:
I tested this a little bit and it seems to work fine. What exactly doesn't work here? If you expect async response for the `addListener()` you can add `return true;` after the `sendResponse()`.

`manifest_json` and `manifest_firefox.json` needs to include `background/offscreen.js`, otherwise `retrieveColorScheme()` will fail.

Reply:
Sorry for the confusion, it definitely works. I just wanted to point out that it is an exception to what is done elsewhere in the codebase.

If I remember correctly this is perfectly good to merge.

And yes, it might be worth removing the polyfill down the line, but that is not a requirement for this change.

Reply:
I just read your comment more closely and fixed the manifest files.